internal/recert/recert.go

```diff
@@ -168,17 +182,28 @@ func CreateRecertConfigFile(seedReconfig *seedreconfig.SeedReconfiguration, seed
 	config.CNSanReplaceRules = []string{
 		fmt.Sprintf("system:node:%s,system:node:%s", seedClusterInfo.SNOHostname, seedReconfig.Hostname),
 		fmt.Sprintf("%s,%s", seedClusterInfo.SNOHostname, seedReconfig.Hostname),
-		fmt.Sprintf("%s,%s", seedClusterInfo.NodeIP, seedReconfig.NodeIP),
 		fmt.Sprintf("api.%s,api.%s", seedFullDomain, clusterFullDomain),
 		fmt.Sprintf("api-int.%s,api-int.%s", seedFullDomain, clusterFullDomain),
 		fmt.Sprintf("*.apps.%s,*.apps.%s", seedFullDomain, clusterFullDomain),
 	}
-	// check if there is an ingress CN provided for backwards compatibility
+
+	if len(seedClusterInfo.NodeIPs) == len(allNodeIPs) {
+		for i := range seedClusterInfo.NodeIPs {
+			config.CNSanReplaceRules = append(
+				config.CNSanReplaceRules, fmt.Sprintf("%s,%s", seedClusterInfo.NodeIPs[i], allNodeIPs[i]),
+			)
+		}
+	}
```

Comment:
What is this?
And what when  `len(seedClusterInfo.NodeIPs) != len(allNodeIPs)` is that a valid scenario?